src/components/BannerLink/BannerLink.tsx

```diff
@@ -0,0 +1,57 @@
+import { alpha, Box, Container, Stack, useTheme } from '@mui/material';
+import { Ctas, type CtaProps } from '../../components/Ctas';
+import { type CommonProps } from 'types/components';
+import { type BannerLinkContentProps, Content } from './Content';
+import LoginImg from '../../assets/svg/Login';
```

Comment:
I think we should abstract the icon to shown and not use a specific one, like you are doing here.

Reply:
Ok, now the component accepts a `decoration` prop of type `Generic` to be rendered above the title.